internals/secrethub/run.go

```diff
@@ -478,7 +479,16 @@ func parseDotEnv(r io.Reader) ([]envvar, error) {
 			valColNo++
 		}
 
+		keyColNo := 1 // one for the current column.
+		for _, r := range parts[0] {
+			if !unicode.IsSpace(r) {
+				break
+			}
+			keyColNo++
```

Comment:
What happens with a tab character?

Reply:
```Go
unicode.IsSpace('\t')
// true
```

`unicode.IsSpace` checks for any whitespace, not just spaces.

Reply:
But is it okay that a tab is counted as only 1 column? My IDE counts it as 4.

Reply:
Ahh, good question.

Does it always count it as 4 or do you also have file types for which it counts them as 2?

Reply:
It depends on the used tab-width. To be honest, I don't really know how to fix this nicely. Maybe we should just keep it at 1.